cmd/cosign/cli/options/verify.go

```diff
@@ -190,18 +188,41 @@ func (o *VerifyDockerfileOptions) AddFlags(cmd *cobra.Command) {
 type VerifyBlobAttestationOptions struct {
 	Key           string
 	SignaturePath string
+	BundlePath    string
+
 	PredicateOptions
+	CheckClaims bool
+
+	SecurityKey         SecurityKeyOptions
+	CertVerify          CertVerifyOptions
+	Rekor               RekorOptions
+	CommonVerifyOptions CommonVerifyOptions
+
+	RFC3161TimestampPath string
 }
 
 var _ Interface = (*VerifyBlobOptions)(nil)
 
 // AddFlags implements Interface
 func (o *VerifyBlobAttestationOptions) AddFlags(cmd *cobra.Command) {
 	o.PredicateOptions.AddFlags(cmd)
+	o.SecurityKey.AddFlags(cmd)
+	o.Rekor.AddFlags(cmd)
+	o.CertVerify.AddFlags(cmd)
+	o.CommonVerifyOptions.AddFlags(cmd)
 
 	cmd.Flags().StringVar(&o.Key, "key", "",
 		"path to the public key file, KMS URI or Kubernetes Secret")
 
 	cmd.Flags().StringVar(&o.SignaturePath, "signature", "",
 		"path to base64-encoded signature over attestation in DSSE format")
+
+	cmd.Flags().StringVar(&o.BundlePath, "bundle", "",
+		"path to bundle FILE")
+
+	cmd.Flags().BoolVar(&o.CheckClaims, "check-claims", true,
+		"whether to check the claims found")
+
+	cmd.Flags().StringVar(&o.RFC3161TimestampPath, "rfc3161-timestamp", "",
```

Comment:
This exists under the common verify options - https://github.com/sigstore/cosign/blob/631a26fb92347230b423bedfbaac84ef177a6fd0/cmd/cosign/cli/options/verify.go#L22-L26

Reply:
Thank you!! I'll rebase and update

Reply:
Hmmm hold on, it doesn't exist there.